packages/xrpl/src/models/ledger/Amendments.ts

```diff
@@ -42,4 +42,13 @@ export default interface Amendments extends BaseLedgerEntry {
    * type, so this value is always 0.
    */
   Flags: 0
+
+  /**
+   * This field is missing on Amendments but is present on all other returned objects.
+   */
+  PreviousTxnID: never
```

Comment:
Would prefer for this to be abstracted into an interface for objects without these two properties.